articles/i2p-docker-services/GithubPagesMirror.md

```diff
@@ -0,0 +1,233 @@
+Mirroring an existing Github Pages site using a Static eepSite
+==============================================================
+
+This is kind of a softball tutorial, it's almost the same as the **[Previous tutorial](BasicStaticeepSite.md):**
+but it's intended specifically to allow you to mirror a Github Pages based site
+using Jekyll to i2p using i2pd and Tor. Of course, this means that the
```

Comment:
I feel this could be worded better, such as 'Mirror a Github Pages based site to i2pd and Tor using Jekyll to i2pd', I'll leave this change up to your decision.

Reply:
Another thing that could be included within the header is the fact that doing this does not hide your Github username or the Github repo that it is generated from.